src/primitives/common/plane_sector.rs

```diff
@@ -57,87 +75,56 @@ impl PlaneSector {
         }
 
         Self {
-            half_plane_left: LinearEquation::from_point_angle(center_2x, angle_start),
-            half_plane_right: LinearEquation::from_point_angle(center_2x, angle_end),
+            origin: center_2x,
+            half_plane_left: OriginLinearEquation::with_angle(angle_start),
+            half_plane_right: OriginLinearEquation::with_angle(angle_end),
             operation,
         }
     }
 
     pub fn contains(&self, point: Point) -> bool {
         // `PlaneSector` uses scaled coordinates for an increased resolution.
```

Comment:
Is it just a 2x scaling or is it more/less? It would help to define what the scale is (at least for me).

Reply:
The scaling will be moved out of this struct in the next PR. I'll try to remember to include some more comments about the scale in that PR.